docs/new-ansible-and-freezes.md

```diff
@@ -0,0 +1,146 @@
+# New Ansible Releases and Freezes
+
+## Preamble
+
+Releasing new Ansible major versions and frozen releases requires some special
+handling.
+For information about the general release process,
+see [Ansible Release Process](release-process.md).
+
+## Setting up for a new major release
+
+<!-- TODO: Write a script, playbook, or antsibull-build subcommand to automate this -->
+
+After the release of `X-1.0.0`, it is necessary to create the directory
+structure for Ansible `X`.
+
+1. Determine the current major version and previous major version
+
+    ``` sh
+    MAJOR_VERSION=10
+    PREVIOUS_MAJOR_VERSION=9
```

Comment:
Maybe change these to `NEXT_MAJOR_VERSION` and `CURRENT_MAJOR_VERSION`?

Reply:
Good point. I'll change those.